### PR TITLE
Remove warning for multiple home urls

### LINKED ISSUE #879: Define home url only once in urls.py
## Description
The following command throws a warning because the url home is defined twice in the urls.py file in the main directory.

## Mocks
image
image

### Update [Required]
- [ ] Code should be changed so that it is only defined once in urls.py

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by yashshah2820**
# Description
Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes #879

# Type of Change:
**Delete irrelevant options.**

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

# Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

**Comment by yashshah2820** [comment]
there is app called cities_light in the project so I have commented that in INSTALLED_APP section

File: .gitignore [hunk 1]
```diff
@@ -87,6 +87,9 @@ target/
 # Virtualenv
 venv
 
+# vs code
+.vscode
```
Comment (Monal5031) [inline .gitignore]: No, only changes related to this PR should be here. Please make another issue for this

File: vms/vms/settings.py [hunk 2]
```diff
@@ -13,33 +13,34 @@
...
 INSTALLED_APPS = (
-    'authentication',
     'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'administrator',
+    'authentication',
```
Comment (Monal5031) [inline vms/vms/settings.py]: unrelated changes, please remove them

File: vms/vms/urls.py [hunk 3]
```diff
@@ -12,7 +12,6 @@
     url(r'^authentication/',
         include('authentication.urls', namespace='authentication')),
     url(r'^event/', include('event.urls', namespace='event')),
-    url(r'^home/', include('home.urls', namespace='home')),
```
Comment (Monal5031) [inline vms/vms/urls.py]: You don't have to remove it, just redirect the `^$` to this `/home/`

**Comment by yashshah2820** [comment]
ok on it

File: .gitignore (hunk 1 above)
Comment (yashshah2820) [reply .gitignore]: okay

**Comment by yashshah2820** [comment]
done the necessary changes

**Comment by yashshah2820** [comment]
I have created a new branch and done the necessary changes PR #946

**Comment by Monal5031** [comment]
Please don't create multiple PRs for a single issue.
